scripts/performance/BenchmarkContraction.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+# Copyright (C) 2022-2024 Advanced Micro Devices, Inc. All rights reserved.
+
+set -eux
+
+# ensure this script is in the cwd
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+output_dir=hiptensor-benchmarks
+build_dir=../../build/bin/
+config_dir=../../test/01_contraction/configs
```

Comment:
Just found that there are some new files added in another commit

`test/01_contraction/configs/bench/bilinear_test_params_rank1.yaml`

Are they going to be used in benchmark scripts?

Reply:
Yes. Benchmarks probably won't cover so much parameter space, but have more emphasis on performance and regression checks.